### PR TITLE
planner: adding missing code of similar definition for test

### PR DESCRIPTION
**Pull request by AilinKid**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/45112

Problem Summary:

### What changed and how does it work?
code complement for https://github.com/pingcap/tidb/pull/45158

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/executor/benchmark_test.go [hunk 1]
```diff
@@ -709,6 +709,7 @@ func markChildrenUsedColsForTest(ctx sessionctx.Context, outputSchema *expressio
 			}
 		}
 		childrenUsed = append(childrenUsed, used)
+		prefixLen += childSchema.Len()
```
Comment (AilinKid) [inline pkg/executor/benchmark_test.go:712]: i guess this line logic is missing compared with `markChildrenUsedCols`

**Review by hawkingrei — APPROVED** [review]

**Review by winoros — APPROVED** [review]

**Pull request merged**